consensus/src/network_tests.rs

```diff
@@ -133,19 +154,52 @@ impl NetworkPlayground {
                     if drop_rpc {
                         continue;
                     }
+
+                    let dst_nodeIds = author_to_nodeIds.get(&dst).unwrap();
+
+                    // Bano: FIXME: We should iterate over dst_nodeIds and send the RPC message
+                    // to each dst_nodeId, but I'm doing it for the first one now because of
+                    // ownership / copying issues when I try to send to multiple
+                    let dst_nodeId = dst_nodeIds.get(0).unwrap();
```

Comment:
it should be something like this?
```
for dst in &dst_nodes {
...
}
```

Reply:
There was an ownership issue in match pattern here, which was resolved.